package.json

```diff
@@ -18,6 +18,8 @@
     "release:version": "changeset version && yarn install --immutable"
   },
   "peerDependencies": {
+    "graphql": "^16.8.1",
```

Comment:
Will this clash with any other GraphQL dependcy in the industry? I am a bit ingronant here, but those work this well with Relay?

Reply:
wondering if we can say 

```
"graphql": "*",
```

or choose a minium version.

Reply:
Not an expert here, who should we loop in on this?